src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/index.html

```diff
@@ -8,9 +9,16 @@
     <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
     <link href="css/site.css" rel="stylesheet" />
 </head>
+
 <body>
     <app>Loading...</app>
 
+    <div id="error-ui">
+        An unhandled error has occurred.
+        <a href="#" class='reload'>Reload</a>
+        <a href="#" class='dismiss'>X</a>
```

Comment:
```suggestion
        <a class="reload">Reload</a>
        <a class="dismiss">🗙</a>
```

Hopefully the `href` attributes aren't needed. If they strictly are, maybe they could just be minimized (i.e., `<a href class=...>`).

Reply:
This change suggestion was meant to apply to both lines 18 and 19. Seems that GitHub's new multi-line comment feature doesn't apply to change suggestions yet.

Reply:
Beauty! All I wanted out of the href was to make it clear to the user that this was a clickable element (pointer on hover, blue text for Reload). Your suggestion of `href` without a value seems to do exactly that.